jbpm-system-test/jbpm-clustered-ejb-timers-test/src/test/java/org/kie/samples/integration/testcontainers/KieServerContainer.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.samples.integration.testcontainers;
+
+import java.io.File;
+import java.time.Duration;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.images.builder.ImageFromDockerfile;
+
+public class KieServerContainer extends GenericContainer<KieServerContainer>{
+
+    private static final String ALIAS = "kie-server";
+    private static final int KIE_PORT = 8080;
+
+    private static Logger logger = LoggerFactory.getLogger(KieServerContainer.class);
+
+    public KieServerContainer(String nodeName, Network network, Map<String,String> args) {
+      super( new ImageFromDockerfile()
+           .withBuildArg("IMAGE_NAME", args.get("IMAGE_NAME"))
+           .withFileFromFile("etc/jbpm-custom.cli", new File("src/test/resources/etc/jbpm-custom-"+nodeName+".cli"))
+           .withFileFromClasspath("etc/kjars", "etc/kjars")
+           .withFileFromClasspath("Dockerfile", "etc/Dockerfile")
+           .withFileFromClasspath("etc/drivers/postgresql-42.2.19.jar", "etc/drivers/postgresql-42.2.19.jar"));
```

Comment:
This can be optimized in the future by downloading it using a plugin and copying to test resources for example. But it is fine for now.

Reply:
These are already in test resources, just need to add them to the on-the-fly image

Reply:
I meant that right now we are committing a jar file to the repo, which is not a source code. Moreover, the version is hard-coded here to `42.2.19` but in the EJB Timer POMs there is `42.2.18`, so there is already a difference. If we downloaded the PostgreSQL driver JAR using a dependency plugin to the test resources directory, we could have everything configured in a POM including the version. But as I said, it may be refined in next PRs.

Reply:
ok I see, I've changed it: using maven-dependency-plugin (copy-dependencies goal) to use that driver in the image. Thanks for pointing out @MarianMacik